_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -117,45 +123,253 @@ This `button` element has the text "search" rendered as an magnifying glass icon
 <button aria-label="Find">search</button>
 ```
 
+#### Passed Example 7 
+
+This button has [visible inner text][] that, according to the [label in name algorithm][], is contained within the [accessible name][].  This example shows why the [label in name algorithm][] uses the [visible inner text][] and not the [visible text content][]: the <p> tags insert whitespace into the result in the former but not the latter.
+
+```html
+<button aria-label="Hello world"><p>Hello</p><p>world</p></button>
+```
+
+#### Passed Example 8
+
+Similar to the previous example.
```

Comment:
Referencing "previous example" is a bit risky since nothing guarantees they will stay in the same order next time we rewrite the rule.

Reply:
I agree there is a risk.  In this case I think that the risk is worth it.   With "previous", I claim that the readability is improved now.  If I remove "previous", I can't think of a way to rewrite it that doesn't make it significantly less readable.